### PR TITLE
feat(project-access): 29050 update cap service info api handling 

### PR DESCRIPTION
**Pull request by korotkovao**
## #2200

Support updated API available in @sap/cds: 7.8.0
Update to use endpoints of kind: 'odata' from each service instead of existing urlPath

### CODE REVIEW

File: packages/project-access/src/project/cap.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ export async function getCapModelAndServices(
         services = services.map((value) => {
             return {
                 name: value.name,
-                urlPath: uniformUrl(value.urlPath),
+                urlPath: uniformUrl(value.endpoints?.find((val) => val.kind === 'odata')?.path ?? value.urlPath),
```
Comment (kjose90) [inline packages/project-access/src/project/cap.ts:159]: Would be nice to improve readability 👍 

```suggestion
const { endpoints, urlPath } = value;
const odataEndpoint = endpoints?.find((endpoint) => endpoint.kind === 'odata');
const endpointPath = odataEndpoint?.path ?? urlPath;
return { ... urlPath: uniformUrl(endpointPath) ...};

```

Comment (korotkovao) [reply packages/project-access/src/project/cap.ts]: Thank you, i have added the change as suggested.

**Review by longieirl — APPROVED** [review]
Changes look good.
Test coverage is good.
Changeset is good.

**Review by kjose90 — APPROVED** [review]
Changes looks good and clear
Changeset & tests ✅ 
Thanks.

File: packages/project-access/test/project/cap.test.ts [hunk 2]
```diff
@@ -78,7 +78,184 @@ describe('Test getCapModelAndServices()', () => {
...
+                        },
+                        {
+                            'name': 'Backslash',
+                            'endpoints': [
+                                {
+                                    'path': '\\odata\\service\\with\\backslash/',
+                                    'kind': 'odata'
+                                }
+                            ]
+                        },
```
Comment (devinea) [inline packages/project-access/test/project/cap.test.ts:114]: Path is a web URI so I don't this backslash is valid.

**Review by devinea — APPROVED** [review]
Code changes are clear. 
changeset ✅

File: packages/project-access/test/project/cap.test.ts (hunk 2 above)
Comment (korotkovao) [reply packages/project-access/test/project/cap.test.ts:114]: Thank you, it's a good point. I might leave this test there for now.

**Pull request merged**